src/components/helpPopup/helpPopup.sass

```diff
@@ -0,0 +1,40 @@
+.popup
```

Comment:
rename all of them based on comments above

Reply:
done